components/bl-avatar-generator/src/helpers.js

```diff
@@ -0,0 +1,223 @@
+import { data as characterOptions } from './avatar-data';
+
+const RANDOM_OPTION = 'Random';
+
+const OptionsKeys = {
```

Comment:
What the difference between `ComponentOptionsKeys` and `OptionsKeys`? Could we join them?